### ISSUE TITLE
Open sourced under which License?

### ISSUE THREAD

**Issue by rk8339811** [ZeroOneZeroR/android_audio_mixer#7]
Hi,

I am planning to use this library in my Android App. Can you please let me know under which license (MIT or Apache etc) this library has been open-sourced?

Thanks.

**Comment by chaitanyagupta** [comment]
Yes, it would be great to get some clarity on the license for this project.

**Comment by nlipelt** [comment]
We would like to use this library.  Is there an expected timeframe on when a license will be known?
